foss4gna_24/big-EO-viz.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>Explain this more or remove. This is just to make sure to render the notebook with the correct jupyter kernel/conda env?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Yes. The kernel name is required to execute the next cell. Removing it.</p>